README.md

```diff
@@ -1,12 +1,55 @@
+# Pettamo
 
+## Overview
+
```

Comment:
This project is submitted for Build Modern Data App Hackathon. Link to live demo is [here](https://pettamo.netlify.app/)